CI/setup-windows-sdk.sh

```diff
@@ -91,11 +95,12 @@ echo "    This could take a long time if it is needed to fetch everything, so fe
 echo "    to go and have a cup of tea (other beverages are available) in the meantime...!"
 echo ""
 
-if [ "${MSYSTEM}" = "MINGW64" ]; then
+
+if [ "${BUILD_BITNESS}" = "64" ]; then
   echo "=== Installing Qt6 Packages ==="
   pacman_attempts=1
   while true; do
-    if /usr/bin/pacman -Su --needed --noconfirm \
+    if /usr/bin/pacman -S --noconfirm \
```

Comment:
The `u` in the old code made `pacman` update it's catalogue and thus existing installed packages, and the `--needed` stopped it from reinstalling the same packages if they were already installed. Is the removal of these intentional - and if so, why?

Reply:
Well -u was attempting a system upgrade each time 3x in the setup script which probably isnt needed as the msys2 does a system upgrade when it installs itself, I think? I remove the --needed in an attempt to NOT have qt-base install harfbuzz as a dependency, in fact I tried several things with no success, and went with just removing and installing the compiled version manually.